README.md

```diff
@@ -171,16 +173,33 @@ mstorage.list(){result, error in
         // do something
     }
 }
-// You can also use a Dictionary object for listing options as follows.
-// The available options are "limit", "after" and "before".
+```
+
+* With options
+
+You can also use a `Dictionary` object for listing options as follows.
+The available options are `limit`, `after` and `before`.
+
+```swift
 mstorage.list(["limit": "25", "after": "<media_id>"]){result, error in
     // do something
 }
 ```
 
-### Delete
+* Search
+
+Search media ids by user metadata
```

Comment:
Maybe we can explain about the parameter in detail on here and media-storage-java.
The sentence would be like "You can also nest a `Dictionary` with `filter` key into the listing options to search by user metadata. "